server/app/routes/api/v1/index.js

```diff
@@ -3,10 +3,15 @@ var router = require('express').Router();
 module.exports = router;
 
 router.use('/users', require('./users'));
+router.use('/teams', require('./teams'));
+router.use('/organizations', require('./organizations'));
 router.use('/annotations', require('./annotations'));
-router.use('/questions', require('./assessments/questions'));
+router.use('/questions', require('./questions/questions'));
+router.use('/responses', require('./questions/responses'));
+router.use('/templates', require('./templates'));
 router.use('/evaluations', require('./assessments/evaluations'));
```

Comment:
annotations are comments on lines of code **for** an assessment taken/filled out by an individual right?